concrete_datastore/admin/admin.py

```diff
@@ -85,13 +86,27 @@ def get_list_filter(self, request):
 
         return get_list_filter
 
+    def export_csv(self, request, queryset):
+        fields = [f.name for f in queryset.model._meta.fields]
```

Comment:
we could use the `meta_model` property `"m_export_fields"`.
```python
from concrete_datastore.concrete.meta import get_meta_definition_by_model_name

model = queryset.model
fields = get_meta_definition_by_model_name(model.__name__).get_property('m_export_fields')
```

Reply:
I'd rather not so we can also export non-meta models.

Reply:
And in the admin, we want all information available ; not only the export fields

Reply:
Ok